crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -185,6 +186,59 @@ impl SelectionPanel {
                     }),
                 );
 
+                let ComponentDescriptor {
+                    archetype_name,
+                    archetype_field_name,
+                    component_name,
+                } = component_descriptor;
+
+                if let Some(archetype_name) = archetype_name {
```

Comment:
Wondering if it's better to explicitly call out the archetype / field being `None` when they're none since it's quite unusual 🤔